packages/devextreme-scss/scss/widgets/fluent/chat/_sizes.scss

```diff
@@ -71,10 +90,15 @@ $chat-messagelist-day-header-first-padding-top: 20px !default;
   $chat-errors-row-gap: 2px !default;
   $chat-error-padding-block: 5px !default;
 
-  $chat-messagelist-empty-icon-box-size: 48px !default;;
+  $chat-messagelist-empty-icon-box-size: 48px !default;
   $chat-messagelist-empty-icon-size: 32px !default;
   $chat-messagelist-empty-icon-margin-bottom: 14px !default;
   $chat-messagelist-empty-row-gap: 2px !default;
   $chat-messagelist-empty-message-font-size: 14px !default;
   $chat-messagelist-empty-prompt-font-size: 12px !default;
+
+  $chat-typingindicator-bubble-height: 2.333333em !default;
+  $chat-typingindicator-bubble-width: 5em !default;
```

Comment:
The same mistake: https://github.com/DevExpress/DevExtreme/pull/28229/files#r1819850668

Reply:
I believe this is a design error: https://github.com/DevExpress/DevExtreme/pull/28229#discussion_r1819854218